content/languages/javascript/authoring.md

```diff
@@ -0,0 +1,283 @@
+---
+kind: tutorial
+languages: [javascript]
+sidebar: "language:javascript"
+prev: /languages/javascript/codewars-test/
+---
+
+# JavaScript: creating and translating a kata
+
+This article is meant as a help for kata authors and translators who would like to create new content in JavaScript. It attempts to explain how to create and organize things in a way conforming to [authoring guidelines](/authoring/guidelines/), shows the most common pitfalls and how to avoid them.
```

Comment:
```suggestion
This article is intended for kata authors and translators who would like to create new content in JavaScript. It attempts to explain how to create and organize things in a way conforming to [authoring guidelines](/authoring/guidelines/), shows the most common pitfalls and how to avoid them.
```